pkg/server/storage_api/ranges_test.go

```diff
@@ -32,14 +34,16 @@ func TestRangesResponse(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 	defer kvserver.EnableLeaseHistoryForTesting(100)()
-	srv := serverutils.StartServerOnly(t, base.TestServerArgs{
-		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(110019),
-	})
+	srv := serverutils.StartServerOnly(t, base.TestServerArgs{})
 	defer srv.Stopper().Stop(context.Background())
 	ts := srv.ApplicationLayer()
 
-	// TODO(#110019): grant a special capability to the secondary tenant
-	// before the endpoint can be accessed.
```

Comment:
I don't think we need to grant special capability as of now since this endpoint return its own tenant's ranges.